fudge-ratio/src/main/webapp/code/time.js

```diff
@@ -1,52 +1,46 @@
 class Timer {
   seconds = 0;
-  minutes = 0;
-  hours = 0;
-
   interval = null;
 
+  constructor(time) {
+    if (time) {
+      const seconds = parseInt(time.substring(6, 8));
+      const minutes = 60 * parseInt(time.substring(3, 5));
+      const hours = 3600 * parseInt(time.substring(0, 2));
+
+      this.seconds = seconds + minutes + hours;
+    }
+  }
+
   // Concatenates a zero to the beginning of a number less than 10.
   padd(num) {
     const padding = num < 10 ? "0" : "";
     return padding + num;
   }
 
-  constructor(time) {
-    if (time) {
-      this.hours = parseInt(time.substring(0, 2));
-      this.minutes = parseInt(time.substring(3, 5));
-      this.seconds = parseInt(time.substring(6, 8));
-    }
+  addSeconds(howMany) {
```

Comment:
This functions will add a arbitrary number of seconds, just checking that the number stays non negative